### ISSUE TITLE
App stopped working in ownCloud 10.0.9

### ISSUE THREAD

**Issue by agross** [cowai/camerarawpreviews#26]
Hi,

First off thank you for creating this app. I'm having trouble getting this to work. I use a [custom docker image](https://github.com/agross/owncloud/blob/master/Dockerfile) that includes the imagick module.

When I open CR2 images in ownCloud web I get this when clicking on the respective HTTP request in dev tools:

```json
{
  "message": "I'm truly sorry, but we were unable to generate a preview for this file",
  "success": false
}
```

When the file list loads I see a lot of GET 404s for URLs like these:

https://oc.example.com/remote.php/dav/files/agross/IMG_5053.cr2?c=c1c404471c543834c7dba5f6db7f5f18&x=64&y=64&forceIcon=0&preview=1

No other errors are logged (not even in owncloud.log)

Previews for JPEGs load just fine.

**Comment by cowai** [comment]
Hi,
Can you check your owncloud logs for camerarawpreviews entries?

**Comment by cowai** [comment]
update:
I see that my app stopped worked in owncloud 10.0.9.
You can either use nextcloud or wait for a fix. That fix may take a long time, since there seems to be bigger changes now between nextcloud and owncloud.

**Comment by agross** [comment]
There's nothing logged on my machine. Thank for researching this.

I'll wait for an updated release, I cannot change the underlying system without possibly breaking my users.

**Comment by cowai** [comment]
Fixed in v0.6. Should be in owncloud marketplace and nextcloud app store by now. Thanks for the bug report. This should never have happened in a minor release of owncloud. Very strange.
